packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -1180,30 +1180,3 @@ test(`zcf.reallocate 3 seats, rights NOT conserved`, async t => {
   });
   t.deepEqual(zcfSeat3.getCurrentAllocation(), { Whatever: moola(0) });
 });
-
-test(`zcf.shutdown - userSeat exits`, async t => {
```

Comment:
Why delete these tests? I think they are testing important things, especially for this commit. Is this covered elsewhere?

Reply:
I just tried to test, and as far as I can tell, these tests work fine with the small change of adding a string to the shutdown call: `  zcf.shutdown('reason');`

Reply:
I thought they were pretty useless since they're using the fakeVatAdmin and the fake vatPowers, but you're right that they do show that some of the seat wiring is connected.